papers/bcr-0004-bc32.md

```diff
@@ -0,0 +1,51 @@
+# The BC32 Data Encoding Format
+## BCR-0004
+
+**© 2020 Blockchain Commons**
+
+Author: Wolf McNally<br/>
```

Comment:
You certainly are the lead author on this, but I've been driving this agenda and will be driving it through standards, so can you add my name last?

Reply:
Absolutely. Added your name to this and BCR-0005.